src/main/java/com/rybka/command/ExportCommand.java

```diff
@@ -0,0 +1,33 @@
+package com.rybka.command;
+
+import com.rybka.config.ExportType;
+import com.rybka.config.MapSearchUtil;
+import com.rybka.config.PropertyInfo;
+import com.rybka.dao.CurrencyHistoryDAO;
+import com.rybka.service.export.CSVExportService;
+import com.rybka.service.export.ConsoleExportService;
+import com.rybka.service.export.JSONExportService;
+import lombok.RequiredArgsConstructor;
+
+import java.util.Map;
+import java.util.Properties;
+
+@RequiredArgsConstructor
+public class ExportCommand implements Command {
+    private final Properties reader;
+    private final ConsoleExportService consoleExportService;
+    private final CSVExportService csvExportService;
+    private final JSONExportService jsonExportService;
+    private final CurrencyHistoryDAO currencyHistoryDAO;
+
+    @Override
+    public void execute() {
+        var exportConfigMap = Map.of(
```

Comment:
It would be better to define map in the constructor, as each time we call method a new map is created.

Reply:
fixed